.github/workflows/tag_image_push_rhel.yml

```diff
@@ -0,0 +1,119 @@
+name: hz-enterprise-operator-rhel-autorelease
+
+on:
+  push:
+    branches:
+      - "!*"
+    tags:
+      - "v4.*"
```

Comment:
@mtyazici I could not see any push command for `latest` tag. What happens if we  release patch release(4.0.4) after minor release(4.1.1)?
For example, Dockerhub release script pushes latest tag seperately:
https://github.com/hazelcast/hazelcast-docker/blob/master/.github/workflows/tag_image_push.yml#L42-L45

Reply:
Currently we have no way of changing latest tag from code. It has to be done manually. I will create a ticket about it and share the results with you.

Reply:
I created the case but it is going to be resolved much later than we anticipated. I will also create an issue about it in this repository.